src/features/player/movement.js

```diff
@@ -164,16 +155,13 @@ export default function handleMovement(player) {
   function attemptMove(direction) {
     const oldPos = store.getState().player.position;
     const newPos = getNewPosition(oldPos, direction);
-    checkInteraction(oldPos, newPos, direction)
-    console.log(newPos)
+    checkInteraction(newPos)
     if (
       observeBoundaries(oldPos, newPos) &&
       observeImpassable(oldPos, newPos) &&
       canMove
     ) {
       dispatchMove(direction, newPos);
-    } else {
-      dispatchMove(direction, oldPos);
```

Comment:
this is what was causing the rotate, opps... just realised that now vim doesn't face the direction of a bump... (i will look at more - hopeful be a 2 min fix)

Reply:
now fixed, bumping into blockers is back